### PR TITLE
Update downloader

### PR DESCRIPTION
**Pull request by Ao-senXiong**
As we discussed strange successful installation but had no effect on the Java code, it turns out old commits change the prefix but did not update the downloader in this repo https://github.com/eisopux/checker-framework-vscode/commit/a0a43c5ef798afb6d66d0ed5c7dda2e60f9bfbcc. I update the downloader in this repo and update vscode version number in this PR. 

The installation should succeed by following the instruction.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ out
 node_modules
 download
 .vscode
+checker-framework-language-server-0.2.0.vsix
```
Comment (wmdietl) [inline .gitignore:5]: Use a wildcard to not make this version specific. Probably also better in a separate PR.

Comment (wmdietl) [reply .gitignore:5]: Note the red icon at the end of the file. If you open a PR make sure you don't see that symbol and add the required newlines at the ends of files. Maybe there is an editor config that you can change.

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "Linters"
   ],
   "engines": {
-    "vscode": "^1.62.0"
+    "vscode": "^1.81.1"
```
Comment (wmdietl) [inline package.json:16]: Is there a way to get dependabot to keep this dependency up-to-date, like it does e.g. here #167?
In any event, this should be a separate PR.

File: checker-framework-languageserver-downloader-0.2.0.jar
Comment (wmdietl) [inline checker-framework-languageserver-downloader-0.2.0.jar:1]: We should make a release in the checker-framework-languageserver-downloader project, probably with version 0.3.0 and then upload a file here that uses that version number and removes the old 0.2.0 file.
Ideally we would not have this binary commit in this repository at all. How about checking out a release of checker-framework-languageserver-downloader and then building it as part of the build of this project?
Then we would only have to update the release number or commit hash of the dependency instead of including a binary file.

**Review by wmdietl — CHANGES_REQUESTED** [review]
Thanks for these updates! I suggest closing this PR and opening three new PRs addressing the comments.

**Comment by Ao-senXiong** [comment]
Thanks, I will update these changes with new PRs

File: checker-framework-languageserver-downloader-0.2.0.jar
Comment (Ao-senXiong) [reply checker-framework-languageserver-downloader-0.2.0.jar:1]: Indeed, let me make the changes in a new PR and we should discuss there.

Comment (Ao-senXiong) [reply checker-framework-languageserver-downloader-0.2.0.jar:1]: @wmdietl Prof. Dietl, will you make a new release on both checker-framework-languageserver-downloader and checker-framework-languageserver? I seem do not have permission to do it in this org. Otherwise, please grant me the write permission to the repo.
